composer.json

```diff
@@ -9,5 +9,10 @@
             "email": "devops@findologic.com"
         }
     ],
-    "require": {}
+    "require": {},
+    "autoload": {
+        "psr-0": {
+            "FINDOLOGIC\\Export": "src/main/php/"
```

Comment:
Namespace must end in `\\`, as [documented for Composer](https://getcomposer.org/doc/04-schema.md#psr-0).

Reply:
Done

Reply:
@mmachatschek For future reference, if resolving a review issue doesn't need extra information in the comment, like an explanation why a decision was made, please refrain from doing commenting something like "Done" and just use the button to resolve the issue. I'm getting a mail for every single comment, and I can't seem to find a notification preference to deal with this.